test/fast/Unit tests/nvm_get_node_from_pkg_json

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+die () { echo "$@" ; exit 1; }
+
+\. ../../../nvm.sh
+
+# TODO add more test cases
+TEST_CASES=(
+'{\n\t"engines": {\n\t\t"node":"1.1.1"\n\t}\n}"' '1.1.1'
```

Comment:
i would assume that whatever spacing is present in a .json file, is verbatim the test case - human readable isn't the goal as much as human understandable :-)